ngx-fudis/projects/ngx-fudis/src/lib/components/dialog/dialog.component.scss

```diff
@@ -1,65 +1,79 @@
 @use '../../foundations/spacing/tokens.scss' as spacing;
 @use '../../foundations/breakpoints/mixins.scss' as breakpoints;
-@use '../../foundations/grid/tokens.scss' as gridTokens;
+
+@mixin mobile-dialog-width {
+  width: calc(spacing.$spacing-dialog-xs - (2 * spacing.$spacing-sm));
+
+  @include breakpoints.breakpoint-max('sm') {
+    padding: spacing.$spacing-md spacing.$spacing-sm;
+  }
+}
 
 .fudis-dialog {
   box-sizing: content-box;
   position: relative;
-  margin: spacing.$spacing-md spacing.$spacing-sm;
-  max-width: calc(100vw - (2 * spacing.$spacing-sm));
+  padding: spacing.$spacing-md spacing.$spacing-xl;
+  /* stylelint-disable-next-line unit-disallowed-list */
+  max-width: calc(100vw - 10rem);
```

Comment:
We cannot use rem values without also including --fudis-rem-multiplier variable here. Without it, this 10rem will result in 160px by default and 100px in e.g. SISU.